modules/Kconfig.renesas_fsp

```diff
@@ -17,3 +17,19 @@ config USE_RA_FSP_DTC
 	bool
 	help
 	  Enable RA FSP DTC driver
+
+config USE_RA_FSP_SCE
+	bool
+	help
+	  Enable RA FSP SCE driver
+
+if USE_RA_FSP_SCE
+
+config HAS_RENESAS_RA_RSIP7
```

Comment:
Where is this referenced?

Reply:
RSIP7 is the old name of RSIP-E51A IP on Renesas RA8 MCU. I have modified to update to the latest name of this IP mentioned in the HWM.